client/src/utils/FetchRoomsResponse.ts

```diff
@@ -25,6 +25,25 @@ export const fetchRoomsResponse = async (roomId: string, userId: string): Promis
 
   const responseContent = await response.text();
   const roomResponse = JSON.parse(responseContent);
-
   return roomResponse;
 };
+
+export async function fetchRoom(): Promise<CreateRoomResponse> {
```

Comment:
Arrow syntax here. Also let's make the function name more accurate about what it is doing `createRoom`